src/components/Card.tsx

```diff
@@ -88,15 +96,14 @@ const cardContainer = (
       'py-5': paddingType === 'normal',
       'py-8': typeof paddingType === 'undefined',
     }),
-    width(
-      thin ? 'sm:!w-thin-card' : 'sm:w-card',
-      thin ? 'tiny:w-thin-mobile' : 'w-mobile-card',
-      { 'w-32': thin }
-    ),
+    !useAppStyles ? (thin ? thinWidthStyles : defaultWidthStyles) : undefined,
     margin({ 'mx-auto': !thin }, 'lg:mx-0'),
-    maxHeight({ 'sm:max-h-card': !onlyWrap, 'max-h-mobile-card': !onlyWrap }),
+    maxHeight({
+      'sm:max-h-card': !onlyWrap && !useAppStyles,
+      'max-h-mobile-card': !onlyWrap && !useAppStyles,
+    }),
     space({ 'space-y-6': !nospace }),
-    margin(spinner ? 'mt-4' : undefined),
+    margin({ 'mt-4': spinner !== null }),
```

Comment:
```TS
margin({'mt-4': spinner})
OR
margin({'mt-4': !!spinner})
```
Should work too

Reply:
First one gets rejected by `yarn build`, but second one works :) Thanks